lib/resources/r4/encounter.yaml

```diff
@@ -573,12 +573,40 @@ fields:
         }
       ],
       "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+    },
+    {
+      "extension": [
+        {
+          "id": "ENCNTR:970558313",
+          "valueString": "Gestational AGE",
+          "url": "custom-attribute-name"
+        },
+        {
+          "valueInteger": 256,
+          "url": "custom-attribute-value"
+        }
+      ],
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+    },
+    {
+      "extension": [
+        {
+          "id": "ENCNTR:3339152",
+          "valueString": "Self-Pay Follow Up Date",
+          "url": "custom-attribute-name"
+        },
+        {
+          "valueDateTime": "2020-07-03T12:00:00.000Z",
+          "url": "custom-attribute-value"
+        }
+      ],
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
     }
   note: >
     <ul>
       <li>custom-attribute-name and custom-attribute-value are both required</li>
       <li>id and valueString for custom-attribute-name are required</li>
-      <li>custom-attribute-value can only be of type string</li>
+      <li>custom-attribute-value can only be of type String, Integer, or DateTime</li>
```

Comment:
I thought Ignite typically capitalizes the data types based on how they are represented in the spec here: https://www.hl7.org/fhir/datatypes.html#CodeableConcept

Reply:
I also think we need periods at the end of these notes.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/825/commits/a5347f8fbcc2881fe55f67f9f40f32a631117096